### ISSUE TITLE
0.46: ag-project-regexp escapes regexps when it appears it doesn't have to

### ISSUE THREAD

**Issue by timvisher** [Wilfred/ag.el#106]
![ag-regexp-bug](https://cloud.githubusercontent.com/assets/34676/13320346/3ff18400-db96-11e5-97d7-2b39fa8e30de.gif)

I think the problem is probably right around here, and that the string shouldn't be escaped if it's a regexp, but I don't have time to debug it right now. Is this an issue for anyone else?

```
In GNU Emacs 24.5.1 (x86_64-apple-darwin14.4.0)
 of 2015-08-29 on timvisher-rjmetrics.home
Configured using:
 `configure --prefix=/usr/local/Cellar/emacs/24.5
 --enable-locallisppath=/usr/local/share/emacs/site-lisp
 --infodir=/usr/local/Cellar/emacs/24.5/share/info/emacs --with-xml2
 --without-dbus --with-gnutls --without-popmail --without-x
 --without-ns'

Important settings:
  value of $LANG: en_US.UTF-8
  locale-coding-system: utf-8-unix

Major mode: Emacs-Lisp

Minor modes in effect:
  diff-auto-refine-mode: t
  magit-auto-revert-mode: t
  auto-revert-mode: t
  global-git-commit-mode: t
  async-bytecomp-package-mode: t
  shell-dirtrack-mode: t
  paredit-mode: t
  eldoc-mode: t
  elisp-slime-nav-mode: t
  winner-mode: t
  savehist-mode: t
  global-whitespace-mode: t
  whitespace-mode: t
  global-hl-line-mode: t
  delete-selection-mode: t
  auto-insert-mode: t
  projectile-global-mode: t
  projectile-mode: t
  ido-vertical-mode: t
  flx-ido-mode: t
  ido-ubiquitous-mode: t
  show-paren-mode: t
  ido-everywhere: t
  tooltip-mode: t
  electric-indent-mode: t
  file-name-shadow-mode: t
  global-font-lock-mode: t
  font-lock-mode: t
  auto-composition-mode: t
  auto-encryption-mode: t
  auto-compression-mode: t
  column-number-mode: t
  line-number-mode: t
  hs-minor-mode: t

Recent messages:
you're loaded, Charnock!
timvisher/map-custom-paredit-keys has been called, CHARNOCK!
timvisher/map-custom-paredit-keys has finished, CHARNOCK!
For information about GNU Emacs and the GNU system, type <f1> C-a.
ls does not support --dired; see `dired-use-ls-dired' for more details.
(No files need saving) [3 times]
Ag finished
Quit
Mark saved where search started [2 times]
https://github.com///blob/master/ag.el#L202

Load-path shadows:
/Users/tvisher/.emacs.d/elpa/align-cljlet-0.3/align-cljlet hides ~/.emacs.d/site-lisp/align-cljlet
~/.emacs.d/timvisher/custom hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/custom
/Users/tvisher/.emacs.d/elpa/org-20150824/ox hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-texinfo hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-texinfo
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-publish hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-publish
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-org hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-org
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-odt hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-odt
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-md hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-md
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-man hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-man
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-latex hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-latex
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-icalendar hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-icalendar
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-html hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-html
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-beamer hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-beamer
/Users/tvisher/.emacs.d/elpa/org-20150824/ox-ascii hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ox-ascii
/Users/tvisher/.emacs.d/elpa/org-20150824/org hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org
/Users/tvisher/.emacs.d/elpa/org-20150824/org-w3m hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-w3m
/Users/tvisher/.emacs.d/elpa/org-20150824/org-version hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-version
/Users/tvisher/.emacs.d/elpa/org-20150824/org-timer hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-timer
/Users/tvisher/.emacs.d/elpa/org-20150824/org-table hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-table
/Users/tvisher/.emacs.d/elpa/org-20150824/org-src hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-src
/Users/tvisher/.emacs.d/elpa/org-20150824/org-rmail hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-rmail
/Users/tvisher/.emacs.d/elpa/org-20150824/org-protocol hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-protocol
/Users/tvisher/.emacs.d/elpa/org-20150824/org-plot hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-plot
/Users/tvisher/.emacs.d/elpa/org-20150824/org-pcomplete hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-pcomplete
/Users/tvisher/.emacs.d/elpa/org-20150824/org-mouse hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-mouse
/Users/tvisher/.emacs.d/elpa/org-20150824/org-mobile hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-mobile
/Users/tvisher/.emacs.d/elpa/org-20150824/org-mhe hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-mhe
/Users/tvisher/.emacs.d/elpa/org-20150824/org-macs hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-macs
/Users/tvisher/.emacs.d/elpa/org-20150824/org-macro hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-macro
/Users/tvisher/.emacs.d/elpa/org-20150824/org-loaddefs hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-loaddefs
/Users/tvisher/.emacs.d/elpa/org-20150824/org-list hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-list
/Users/tvisher/.emacs.d/elpa/org-20150824/org-irc hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-irc
/Users/tvisher/.emacs.d/elpa/org-20150824/org-install hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-install
/Users/tvisher/.emacs.d/elpa/org-20150824/org-inlinetask hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-inlinetask
/Users/tvisher/.emacs.d/elpa/org-20150824/org-info hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-info
/Users/tvisher/.emacs.d/elpa/org-20150824/org-indent hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-indent
/Users/tvisher/.emacs.d/elpa/org-20150824/org-id hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-id
/Users/tvisher/.emacs.d/elpa/org-20150824/org-habit hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-habit
/Users/tvisher/.emacs.d/elpa/org-20150824/org-gnus hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-gnus
/Users/tvisher/.emacs.d/elpa/org-20150824/org-footnote hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-footnote
/Users/tvisher/.emacs.d/elpa/org-20150824/org-feed hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-feed
/Users/tvisher/.emacs.d/elpa/org-20150824/org-faces hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-faces
/Users/tvisher/.emacs.d/elpa/org-20150824/org-eshell hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-eshell
/Users/tvisher/.emacs.d/elpa/org-20150824/org-entities hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-entities
/Users/tvisher/.emacs.d/elpa/org-20150824/org-element hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-element
/Users/tvisher/.emacs.d/elpa/org-20150824/org-docview hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-docview
/Users/tvisher/.emacs.d/elpa/org-20150824/org-datetree hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-datetree
/Users/tvisher/.emacs.d/elpa/org-20150824/org-ctags hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-ctags
/Users/tvisher/.emacs.d/elpa/org-20150824/org-crypt hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-crypt
/Users/tvisher/.emacs.d/elpa/org-20150824/org-compat hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-compat
/Users/tvisher/.emacs.d/elpa/org-20150824/org-colview hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-colview
/Users/tvisher/.emacs.d/elpa/org-20150824/org-clock hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-clock
/Users/tvisher/.emacs.d/elpa/org-20150824/org-capture hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-capture
/Users/tvisher/.emacs.d/elpa/org-20150824/org-bibtex hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-bibtex
/Users/tvisher/.emacs.d/elpa/org-20150824/org-bbdb hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-bbdb
/Users/tvisher/.emacs.d/elpa/org-20150824/org-attach hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-attach
/Users/tvisher/.emacs.d/elpa/org-20150824/org-archive hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-archive
/Users/tvisher/.emacs.d/elpa/org-20150824/org-agenda hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/org-agenda
/Users/tvisher/.emacs.d/elpa/org-20150824/ob hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-tangle hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-tangle
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-table hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-table
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-sqlite hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-sqlite
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-sql hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-sql
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-shen hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-shen
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-screen hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-screen
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-scheme hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-scheme
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-scala hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-scala
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-sass hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-sass
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-ruby hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-ruby
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-ref hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-ref
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-R hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-R
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-python hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-python
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-plantuml hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-plantuml
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-picolisp hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-picolisp
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-perl hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-perl
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-org hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-org
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-octave hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-octave
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-ocaml hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-ocaml
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-mscgen hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-mscgen
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-maxima hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-maxima
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-matlab hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-matlab
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-makefile hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-makefile
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-lob hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-lob
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-lisp hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-lisp
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-lilypond hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-lilypond
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-ledger hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-ledger
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-latex hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-latex
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-keys hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-keys
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-js hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-js
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-java hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-java
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-io hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-io
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-haskell hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-haskell
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-gnuplot hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-gnuplot
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-fortran hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-fortran
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-exp hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-exp
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-eval hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-eval
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-emacs-lisp hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-emacs-lisp
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-dot hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-dot
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-ditaa hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-ditaa
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-css hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-css
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-core hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-core
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-comint hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-comint
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-clojure hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-clojure
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-calc hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-calc
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-C hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-C
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-awk hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-awk
/Users/tvisher/.emacs.d/elpa/org-20150824/ob-asymptote hides /usr/local/Cellar/emacs/24.5/share/emacs/24.5/lisp/org/ob-asymptote

Features:
(shadow sort mail-extr emacsbug sendmail misearch multi-isearch vc-hg
vc-git ag vc-svn find-dired s smex linum magit-blame magit-stash
magit-bisect magit-remote magit-commit magit-sequence magit magit-apply
magit-wip magit-log magit-diff smerge-mode diff-mode magit-core
magit-autorevert autorevert filenotify magit-process magit-popup
magit-mode magit-git crm magit-section magit-utils git-commit log-edit
message rfc822 mml mml-sec mm-decode mm-bodies mm-encode mail-parse
rfc2231 rfc2047 rfc2045 ietf-drums mailabbrev mail-utils gmm-utils
mailheader pcvs-util with-editor async-bytecomp async tramp-sh tramp
tramp-compat auth-source gnus-util mm-util mail-prsvr password-cache
tramp-loaddefs trampver shell pcomplete format-spec xterm paredit eldoc
elisp-slime-nav help-mode hideshow php-mode speedbar sb-image ezimage
image dframe etags flymake compile comint ansi-color add-log cc-mode
cc-fonts cc-guess cc-menus cc-cmds cc-styles cc-align cc-engine cc-vars
cc-defs regexp-opt mdfind solarized-light-theme solarized-definitions cl
sensitive sequences disp-table winner ring savehist avoid whitespace
hl-line delsel autoinsert cus-start cus-load server bookmarklets dired-x
dired projectile ibuf-ext ibuffer dash thingatpt ido-vertical-mode
flx-ido flx ido-ubiquitous ido-completing-read+ easy-mmode cl-macs gv
align-cljlet-autoloads better-defaults-autoloads time-date paren edmacro
kmacro saveplace ido clojure-test-mode-autoloads advice help-fns eieio
byte-opt bytecomp byte-compile cl-extra cl-loaddefs cl-lib cconv
eieio-core markdown-mode-autoloads marmalade-autoloads pbcopy-autoloads
s-autoloads sensitive-autoloads sequences-autoloads wgrep-autoloads info
tool-bar easymenu package epg-config tooltip electric uniquify
ediff-hook vc-hooks lisp-float-type tabulated-list newcomment lisp-mode
prog-mode register page menu-bar rfn-eshadow timer select mouse jit-lock
font-lock syntax facemenu font-core frame cham georgian utf-8-lang
misc-lang vietnamese tibetan thai tai-viet lao korean japanese hebrew
greek romanian slovak czech european ethiopic indian cyrillic chinese
case-table epa-hook jka-cmpr-hook help simple abbrev minibuffer nadvice
loaddefs button faces cus-face macroexp files text-properties overlay
sha1 md5 base64 format env code-pages mule custom widget
hashtable-print-readable backquote make-network-process multi-tty emacs)

Memory information:
((conses 16 356704 223623)
 (symbols 48 33575 0)
 (miscs 40 382 1191)
 (strings 32 72462 135145)
 (string-bytes 1 2225923)
 (vectors 16 44993)
 (vector-slots 8 1243545 21374)
 (floats 8 6467 147)
 (intervals 56 1647 1448)
 (buffers 960 17))

```

**Comment by Wilfred** [comment]
This sounds like a duplicate of #94. Are you using the version from MELPA stable?

**Comment by timvisher** [comment]
I am. Maybe cut a new release?

**Comment by Wilfred** [comment]
OK, I've released 0.47. Thanks for bringing this to my attention :)

**Comment by timvisher** [comment]
Lookin' good! :)
